src/Controller/MainController.java

```diff
@@ -93,63 +93,66 @@ public static void setSpc(StudyPlannerController newSpc) {
 	public static void initialise() {
 		try {
 			ui.showStartup();
-			// If a file is present:
-			if (plannerFile.exists()) {
-				Cipher cipher = Cipher.getInstance("Blowfish");
-				cipher.init(Cipher.DECRYPT_MODE, key64);
-				try (CipherInputStream cis = new CipherInputStream(
+			Cipher cipher = Cipher.getInstance("Blowfish");
+			cipher.init(Cipher.DECRYPT_MODE, key64);
+			try (CipherInputStream cis = new CipherInputStream(
 						new BufferedInputStream(new FileInputStream(plannerFile)), cipher);
-						ObjectInputStream ois = new ObjectInputStream(cis)) {
+					ObjectInputStream ois = new ObjectInputStream(cis)) {
 
+				// If a file is present:
+				if (plannerFile.exists()) {
 					SealedObject sealedObject = (SealedObject) ois.readObject();
 					spc = new StudyPlannerController((StudyPlanner) sealedObject.getObject(cipher));
 
 					// Sample note
 					if (spc.getPlanner().getCurrentStudyProfile() != null && spc.getPlanner()
 							.getCurrentStudyProfile().getName().equals("First year Gryffindor")) {
 						UIManager.reportSuccess(
-								"Note: This is a pre-loaded sample StudyPlanner, as used by Harry "
-								+ "Potter. To make your own StudyPlanner, restart the application "
-								+ "and choose \"New File\".");
+							"Note: This is a pre-loaded sample StudyPlanner, as used by Harry "
```

Comment:
The indentation of these lines should not have changed.

Reply:
this is because the check style told me to do that. I will fix it